3DAmsterdam/Assets/Amsterdam3D/Scripts/BAG-API/ImportWKBP.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Networking;
+
+public class ImportWKBP : ImportAPI
+{
+    public static ImportWKBP Instance = null;
+    public WKBP.RootBeperkingen wkbpData;
+
+    private void Awake()
+    {
+        // maak een singleton zodat je deze class contant kan aanroepen vanuit elke hoek
+        if (Instance == null)
+        {
+            Instance = this;
+        }
+    }
+
+    public IEnumerator CallWKBP(Pand.PandResults tempVerblijf)
+    {
+        string wkbpURL = "https://api.data.amsterdam.nl/wkpb/beperking/?verblijfsobjecten__id=" + tempVerblijf.verblijfsobject.verblijfsobjectidentificatie;
```

Comment:
ook hier de base url liever in een variabele

Reply:
Is opgelost